gnosis_vpn-lib/src/session/mod.rs

```diff
@@ -18,6 +18,8 @@ mod protocol;
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct Session {
```

Comment:
```suggestion
#[serde(rename_all = "camelCase")]
pub struct Session {
```

See more in https://serde.rs/attr-rename.html